packages/core/src/utilities/imageToWorldCoords.ts

```diff
@@ -0,0 +1,64 @@
+import { vec3 } from 'gl-matrix';
+import { metaData } from '..';
+import { Point2, Point3 } from '../types';
+
+type Options = {
+  imageId?: string;
+};
+
+/**
+ * Given the 2d coordinates on the image space with [0,0] being the top left corner
+ * of the top left pixel, and options which includes the imageId, it returns the
+ * 3d coordinates on the world space.
+ * @param imageCoords - The 2d coordinates on the image
+ * @param options - options which includes the imageId
+ * @returns The 3d coordinates on the world.
+ *
+ */
+export default function imageToWorldCoords(
+  imageCoords: Point2,
+  options: Options
+): Point3 | undefined {
+  const { imageId } = options;
```

Comment:
It feels weird to me that this is in options, things in options I expect to have a default or suchlike.

You will always need an imageId for this method, so I think it should be (imageId, imageCoords)?